frontend/www/js/omegaup/components/Countdown.test.ts

```diff
@@ -1,20 +1,52 @@
 import { shallowMount } from '@vue/test-utils';
 
 import omegaup_Countdown from './Countdown.vue';
+let dateNowSpy: any;
 
 describe('Countdown.vue', () => {
+  beforeAll(() => {
+    const now = Date.now();
+    // Lock Time
+    dateNowSpy = jest.spyOn(Date, 'now').mockImplementation(() => now);
+  });
+
+  afterAll(() => {
+    // Unlock Time
+    dateNowSpy.mockRestore();
```

Comment:
```suggestion
    dateNowSpy!.mockRestore();
```

aquí se vale usar una aseveración de no-nulable.

Reply:
Ah, no había visto este comentario hasta después de hacer el push. Tampoco sabía que se podía hacer eso,

Reply:
Ahh, olvídalo. Al linter no le gustó la aseveración no-nulable:
```
frontend/www/js/omegaup/components/Countdown.test.ts:13:5: 
[eslint] Forbidden non-null assertion. [@typescript-eslint/no-non-null-assertion]
        dateNowSpy!.mockRestore();
        ^^^^^^^^^^^
```